detekt_custom.yml

```diff
@@ -815,7 +818,9 @@ datadog:
       - "kotlin.collections.arrayListOf()"
       - "kotlin.collections.ArrayList(kotlin.collections.MutableCollection?)"
       - "kotlin.collections.Collection.flatten()"
+      - "kotlin.collections.Collection.forEach(kotlin.Function1)"
       - "kotlin.collections.Collection.isNotEmpty()"
+      - "kotlin.collections.listOf()"
```

Comment:
need to order it properly, same for the lines below.

Reply:
removed.